clusterloader2/testing/watch-list/config.yaml

```diff
@@ -0,0 +1,62 @@
+{{$testDuration := "5m"}}
+name: huge-secrets
+namespace:
+  number: 1
+  prefix: "huge-secrets"
+tuningSets:
+- name: Uniform10qps
+  qpsLoad:
+    qps: 10
+steps:
+- name: Create secrets
+  phases:
+  - namespaceRange:
+      min: 1
+      max: 1
+      basename: huge-secrets
+    replicasPerNamespace: 400
+    tuningSet: Uniform10qps
+    objectBundle:
+    - basename: huge-secret
+      objectTemplatePath: "secret.yaml"
+- name: Start measurements
+  measurements:
+    - Identifier: TestMetrics
+      Method: TestMetrics
+      Params:
+        action: start
```

Comment:
I would also add verification of api call latency

https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/testing/load/modules/measurements.yaml#L52-L60

Reply:
BTW will the verification fail if the api calls will be slow/above some threshold?